stubs/openpyxl/openpyxl/worksheet/worksheet.pyi

```diff
@@ -24,106 +31,106 @@ class Worksheet(_WorkbookChild):
     PAPERSIZE_A5: str
     ORIENTATION_PORTRAIT: str
     ORIENTATION_LANDSCAPE: str
-    def __init__(self, parent, title: Incomplete | None = ...) -> None: ...
+    def __init__(self, parent: Workbook, title: str | None = ...) -> None: ...
     @property
-    def sheet_view(self): ...
+    def sheet_view(self) -> SheetView: ...
     @property
-    def selected_cell(self): ...
+    def selected_cell(self) -> Cell: ...
     @property
-    def active_cell(self): ...
+    def active_cell(self) -> Cell: ...
     @property
-    def page_breaks(self): ...
+    def page_breaks(self) -> PageBreak: ...
```

Comment:
This is incorrect; it returns a tuple of some kind: https://foss.heptapod.net/openpyxl/openpyxl/-/blob/branch/3.0/openpyxl/worksheet/worksheet.py#L157